packages/overlay/src/overlay-stack.ts

```diff
@@ -48,6 +48,10 @@ interface ManagedOverlayContent {
     updateComplete?: Promise<boolean>;
 }
 
+function nextFrame(): Promise<void> {
```

Comment:
Any reason not to import `nextFrame` from '@open-wc/testing'?

Reply:
I didn't want to set a standard for importing "test" code into the production library...but maybe that's being overly strict?

Reply:
👍 no worries, they're identical today but we can't expect a testing helper to always be production-ready. And it's 1 line.